ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_creation.py

```diff
@@ -489,3 +489,41 @@ def test_paddle_meshgrid(
         on_device=on_device,
         **args,
     )
+
+
+# triu_indices
+@handle_frontend_test(
+    fn_tree="paddle.triu_indices",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer")
+    ),
```

Comment:
No need to keep this here since you are not using `x` anywhere as you're generating the inputs `col`, `row` and `offset` explicitly